packages/lib-react-components/src/MetaToolsButton/MetaToolsButton.js

```diff
@@ -17,16 +17,21 @@ export const StyledPlainButton = styled(PlainButton)`
 function MetaToolsButton (props) {
   const { disabled, icon, onClick, text } = props
 
-  return (
-    <StyledPlainButton
-      disabled={disabled}
-      icon={icon}
-      labelSize='small'
-      text={text}
-      onClick={onClick}
-      {...props}
-    />
-  )
+  try {
+    return (
+      <StyledPlainButton
+        data-testid="test-meta-tools-button"
+        disabled={disabled}
+        icon={icon}
+        labelSize='small'
+        text={text}
+        onClick={onClick}
+        {...props}
+      />
+    )
```

Comment:
Question: why is this render wrapped in a `try`? I don't think our other components run try-catches in their render, so I'm assuming this is for a debug?

Also, if we ever have a try-catch in the main render method (and the _try_ returns elements to render), then I assume the _catch_ would also need to return _something,_ even if it's `return null`